.github/workflows/build.yml

```diff
@@ -150,6 +150,10 @@ jobs:
             targetPlatform: Android
             vrsdk: Oculus
             extraoptions: -setDefaultPlatformTextureFormat astc -btb-il2cpp -btb-experimental
+          - name: Pico
+            targetPlatform: Android
+            vrsdk: Pico
+            extraoptions: -setDefaultPlatformTextureFormat astc -btb-il2cpp
```

Comment:
Do we also want to do an experimental build?

Reply:
Sure, though I'm guessing there may not be a huge market for it right now!